apps/cow-tools/src/lib/priceCheckers.ts

```diff
@@ -19,7 +19,8 @@ export const priceCheckerInfoMapping = {
         type: "uint256",
         label: "Token to buy minimum amount",
         inputType: "number",
-        convertInput: (input: number) => BigInt(input * 1e18), // TODO: BLEU-333
+        convertInput: (input: number, decimals: number) =>
+          BigInt(input * 10 ** decimals),
```

Comment:
want to check if this is right on the enconding/decoding. My thinking is that normal number to number w decimals are `number * 10 ** decimals`